docs/static/releasenotes.asciidoc

```diff
@@ -37,6 +38,63 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+[[logstash-6-8-6]]
+* bump version to 6.8.6 https://github.com/elastic/logstash/pull/11327[#11327]
+* Support remove_field on metadata and tests https://github.com/elastic/logstash/pull/11334[#11334]
+* [DOCS] Fixes monitoring link https://github.com/elastic/logstash/pull/11341[#11341]
+* [DOCS] Replaces occurrences of xpack-ref https://github.com/elastic/logstash/pull/11366[#11366]
+* [DOCS] Added older release notice. https://github.com/elastic/logstash/pull/11409[#11409]
+
+==== Plugins
+
+*Fluent Codec*
+
+* Handle EventTime msgpack extension to handle nanosecond precision time and add its parameter [#18](https://github.com/logstash-plugins/logstash-codec-fluent/pull/18)
+
+*Elasticsearch Filter*
+
+* Loosen restrictions on Elasticsearch gem ([#120](https://github.com/logstash-plugins/logstash-filter-elasticsearch/pull/120))
+
+*Grok Filter*
+
+*  Added: support for timeout_scope [#153](https://github.com/logstash-plugins/logstash-filter-grok/pull/153)
+
+*Elasticsearch Input*
+
+* Loosen restrictions on Elasticsearch gem [#110](https://github.com/logstash-plugins/logstash-input-elasticsearch/pull/110)
+
+*Gelf Input*
+
+* Updated library to gelfd2 [#48](https://github.com/logstash-plugins/logstash-input-gelf/pull/48)
+
+*Http Input*
+
+* Update netty and tcnative dependency [#118](https://github.com/logstash-plugins/logstash-input-http/issues/118)
+
+* Added 201 to valid response codes [#114](https://github.com/logstash-plugins/logstash-input-http/issues/114)
+* Documented response\_code option
+
+*Jdbc Input*
+
+* Fixed issue where paging settings in configuration were not being honored [#361](https://github.com/logstash-plugins/logstash-input-jdbc/pull/361)
+
+*Rabbitmq Input*
+
+* Docs: Optional queue args: link to RabbitMQ docs instead
+
+*Snmp Input*
+
```

Comment:
@yaauie Was pushing a minor update to this plugin for the patch release intentional? https://github.com/elastic/logstash/blob/6.8/Gemfile.jruby-2.5.lock.release#L375